base_user_role/wizard/create_from_user.py

```diff
@@ -0,0 +1,46 @@
+from odoo import models, fields
+
+
+class CreateFromUserWizard(models.TransientModel):
+    _name = "base_user_role.create_from_user_wizard"
+    _description = "Create role from user wizard"
+
+    name = fields.Char("Name")
+    assign_to_user = fields.Boolean("Assign to user", default=True)
+
```

Comment:
```suggestion
    user_id = fields.Many2one(
        comodel_name="res.users",
        required=True,
        readonly=True,
        default=lambda x: x._default_user_id()
    )
    
    def _default_user_id(self):
        user_ids = self.env.context.get("active_ids", [])
        if user_ids:
             return user_ids[0]
        return False

```
This change allow to display the field in the view of the wizard.

Reply:
I'd rather skip this one if you are ok with this. As far as I can tell it is not common for wizards that apply on selected records to display the records they are being applied on. On the other hand, the simplification in `create_from_user` is not significant IMHO, and comes at the cost of these several suggested additional lines of code.